### PR TITLE
fix: generate the Kptfile for pipeline catalogs

### PR DESCRIPTION
**Pull request by jstrachan**
so that we can upgrade them later on via `jx gitops upgrade`

### CODE REVIEW

File: pkg/cmd/importcmd/import.go [hunk 1]
```diff
@@ -544,173 +543,6 @@ func (o *ImportOptions) SetReporter(reporter ImportReporter) {
 	o.reporter = reporter
 }
 
-// HasJenkinsfile returns  the ile name if there is a Jenkinsfile or empty string if there is not
-func (o *ImportOptions) HasJenkinsfile() (string, error) {
```
Comment (hervelemeur) [inline pkg/cmd/importcmd/import.go:548]: Shouldn't the block https://github.com/jenkins-x/jx-project/blob/d780df95daf894574244a9f9f9f1e101b61c1ce8/pkg/cmd/importcmd/import.go#L384-L387 then need to be commented/deleted too?

File: pkg/cmd/importcmd/import.go [hunk 2]
```diff
@@ -407,13 +407,25 @@ func (o *ImportOptions) Run() error {
 	}
 	*/
 
-	gitOpsRepo, err := IsGitOpsRepositoryWithPipeline(o.Dir)
+	// lets disable the build pack if we have a jenkins-x.yml or a .lighthouse/*/triggers.yaml file
+	jxProjectFile := filepath.Join(o.Dir, config.ProjectConfigFileName)
```
Comment (hervelemeur) [inline pkg/cmd/importcmd/import.go:411]: I guess [the PR](https://github.com/jenkins-x/jx-project/pull/86) can be closed then ^^

File: pkg/cmd/importcmd/import.go (hunk 1 above)
Comment (jstrachan) [reply pkg/cmd/importcmd/import.go:548]: its not really being used yet - but I'm hoping to add back in the Jenkins import logic so if there's a Jenkinsfile we ask folks if they want to import the source into a Jenkins server

Comment (jstrachan) [reply pkg/cmd/importcmd/import.go:548]: its slightly more complex though; as you may wish to 

* ignore the Jenkinsfile completely in jx as its being used by a separate Jenkins pipeline in a separate Jenkins server 
* import the project into a Jenkins server inside jenkins x (or outside): https://jenkins-x.io/docs/v3/guides/jenkins/getting-started/
* use Jenkinsfile runner via tekton

File: pkg/cmd/importcmd/import.go (hunk 2 above)
Comment (jstrachan) [reply pkg/cmd/importcmd/import.go:411]: thanks for helping though! :)

**Pull request merged**